actionview/lib/action_view/layouts.rb

```diff
@@ -405,11 +405,11 @@ def _normalize_layout(value)
     #
     # ==== Parameters
     # * <tt>formats</tt> - The formats accepted to this layout
-    # * <tt>require_layout</tt> - If set to true and layout is not found,
-    #   an +ArgumentError+ exception is raised (defaults to false)
+    # * <tt>require_layout</tt> - If set to +true+ and layout is not found,
+    #   an +ArgumentError+ exception is raised (defaults to +false+)
```

Comment:
Agree Andrew, in practice it's a convention that has not worked very well. 

Also, in code examples there is even no typography so historically some methods have needed to change their API because of examples like

```ruby
obj.predicate? # => true
```

I brought this topic to discussion a while back in core, I think it needs to be addressed somehow, but we didn't reach any agreement.